README.md

```diff
@@ -418,8 +418,8 @@ spec:
     name: venafi-tpp-issuer
     kind: Issuer
 ```
-
-We can apply this configuration to the cluster using:
+First we have to open the `pingpong.yaml` file, add the URL to your Venafi instance and save it again.
```

Comment:
May be worth adding a comment explaining what needs replacing/editing/where to add this?